packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -449,36 +452,51 @@ export const InputBoxComponent = (props: InputBoxComponentProps): JSX.Element =>
       selectionEndValue: number | null
     ): void => {
       /* @conditional-compile-remove(mention) */
```

Comment:
nit pick: Do we need a conditional compile flag here? Since it looks like the whole `const handleOnSelect` is already being flagged.

Reply:
No, will pull